pkg/cli/cmds/server.go

```diff
@@ -75,8 +75,7 @@ var (
 		"agent-token-file":           nil,
 		"server":                     nil,
 		"secrets-encryption": {
```

Comment:
Should this just be nil? It's probably not obvious from the code what is going on here.  `nil` just means "take the flag as is."  The intention here is that rke2 hides/removes/changes flags of k3s and we wanted it such that if k3s changed a flag rke2 would panic if it didn't have an explicit action for the flag.  The fear is that k3s adds a flag that is not appropriate for rke2 and we didn't notice and slips through the cracks.  So `nil` is forcing us to explicitly confirm the flag, but it takes not action to modify it.  To make the code clearer possibly a `const copy = nil` could be created and we replace `nil` with `copy`.